modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/TargetPlatformIDEPlugin.java

```diff
@@ -217,4 +227,38 @@ private void _configureEclipseModel(
 		plusConfigurations.add(targetPlatformIDEConfiguration);
 	}
 
+	private void _configureIdeaModel(
+		Project project, Configuration targetPlatformIDEConfiguration) {
+
+		IdeaModel ideaModel = GradleUtil.getExtension(project, IdeaModel.class);
+
+		IdeaModule ideaModule = ideaModel.getModule();
+
+		Map<String, Map<String, Collection<Configuration>>> scopes =
+			ideaModule.getScopes();
+
+		GeneratedIdeaScope generatedIdeaScope = GeneratedIdeaScope.PROVIDED;
```

Comment:
You don't need this variable, you can chain with `GeneratedIdeaScope.PROVIDED.name()`.